locales/en-US.yml

```diff
@@ -138,6 +138,8 @@ cacheRemoteFilesDescription: "When this setting is disabled, remote files are lo
 flagAsBot: "Mark this account as as bot"
 flagAsBotDescription: "Enable this option if this account is controlled by a program. If enabled, it will act as flag for other developers to prevent endless interaction chains with other bots and adjust Misskey's internal systems to treat this account as a bot."
 flagAsCat: "Mark this account as a cat"
+flagPleroma: "Show replies in timeline"
+flagPleromaDescription: "Show users replying to other users in the timeline"
```

Comment:
I think the ja-JP file is used as the base for translations so these lines should be added there too.

Reply:
This is correct. Keys must be included in ja-JP.

Reply:
Thanks! I just copied them over as-is to the JP file, though, as I don't know a word of Japanese.

Reply:
See below comment, i've suggested japanese strings.